nio_cli/commands/buildspec.py

```diff
@@ -0,0 +1,96 @@
+import json, os, sys
+from nio.util.discovery import is_class_discoverable as _is_class_discoverable
+from niocore.core.loader.discover import Discover
+from .base import Base
+
+
+def is_class_discoverable(_class, default_discoverability=True):
+    return _is_class_discoverable(_class, default_discoverability)
+
+
+class BuildSpec(Base):
+
+    def __init__(self, options, *args, **kwargs):
+        super().__init__(options, *args, **kwargs)
+        self._repo = self.options['<repo-name>']
+
+    def run(self):
+        spec = {}
+        from nio.block.base import Block
```

Comment:
I was having some weird issues when it was in the other spot. I'll try again and at least add a comment to the code so the next person with this question knows why.